src/store/actions.ts

```diff
@@ -354,7 +361,7 @@ export const confirmAction: ActionIF = async ({ commit }, action: string): Promi
 
 export const findNameRequest: ActionIF = async ({ commit, getters }): Promise<void> => {
```

Comment:
This looks like something else that should be a higher-level function (not in actions!).